scripts/cat_relations/interaction.py

```diff
@@ -295,8 +176,8 @@ def create_interaction(inter_list) -> list:
         created_list.append(
             SingleInteraction(
                 interact_id=inter["id"],
-                biome=inter["biome"] if "biome" in inter else ["Any"],
-                season=inter["season"] if "season" in inter else ["Any"],
+                biome=inter["biome"] if "biome" in inter else ["any"],
```

Comment:
```suggestion
                biome=inter.get("biome", ["any"]),
```
this is just a genuine suggestion and question - is there a reason we don't use `get` in these or is it just muscle memory?

Reply:
I couldn't tell you, this wasn't smth I wrote (my change here was just killing `Any` via find-replace).  I don't see any reason we couldn't use `get` though